nion/swift/Panel.py

```diff
@@ -420,27 +541,12 @@ def _repaint(self, drawing_context: DrawingContext.DrawingContext) -> None:
                     drawing_context.stroke()
 
             if self.__display_close_control:
-                with drawing_context.saver():
-                    drawing_context.begin_path()
-                    close_box_left = canvas_size.width - (20 - 7)
-                    close_box_right = canvas_size.width - (20 - 13)
-                    close_box_top = canvas_size.height // 2 - 3
-                    close_box_bottom = canvas_size.height // 2 + 3
-                    drawing_context.move_to(close_box_left, close_box_top)
-                    drawing_context.line_to(close_box_right, close_box_bottom)
-                    drawing_context.move_to(close_box_left, close_box_bottom)
-                    drawing_context.line_to(close_box_right, close_box_top)
-                    drawing_context.line_width = 1.5
-                    drawing_context.line_cap = "round"
-                    drawing_context.stroke_style = self.__control_style
-                    drawing_context.stroke()
-
-            with drawing_context.saver():
-                drawing_context.font = self.__font
-                drawing_context.text_align = 'center'
-                drawing_context.text_baseline = 'bottom'
-                drawing_context.fill_style = '#000'
-                drawing_context.fill_text(self.title, canvas_size.width // 2, canvas_size.height - self.__text_offset)
+                if platform.system() == "Darwin":  # Mac
+                    self._draw_controls_mac(drawing_context)
+                elif platform.system() == "Linux":
+                    self._draw_controls_linux(drawing_context)
+                else:
+                    self._draw_controls_windows(drawing_context)
```

Comment:
These all fall back to drawing `_draw_controls_windows` - is there any reason to make them different for this particular PR? We could add platform specific code in the next PR. I'm ok leaving this, but just trying to clean up unnecessary changes. What do you think?

Reply:
You're correct they all default currently to the same (windows) rendering mode. I unrolled this to do the If/else bit as part of the tidying up bit.  I don't mind removing it, but the plan was to try and work towards the other half of the Issue of having the more platform based titles.  So it is a structural part of that, with each platform going down its own rendering path of placing the controls but re-using the grouping code (edit to the right of the title etc).

Reply:
Do you plan to do that for this PR or do you want me to merge already?

I'm not too picky on this, but generally I think it's better practice to make each commit stand on its own without anything extraneous. This 'commit' (after we squash the 3 commits into a single commit) adds the pencil icon. The additional platform stuff is extraneous with respect to that change, so I would generally leave it out. The follow-up commit/PR will be something like "add platform specific variants for headers" and would include the platform specific changes.

Reply:
I've pulled the 'styling' parts out as I think you are right about not leaving the placeholders in there if we aren't working on them right now, and the more I have thought about it the more there are probably a lot more questionmarks and open ended questions on how styling wants to be done that are more far reaching than this relatively small Issue/PR.